eth/downloader/downloader.go

```diff
@@ -25,7 +25,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	ethereum "github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum"
```

Comment:
Please don't change this here if you don't do anything else with the downloader.